reference/api/search.md

```diff
@@ -473,46 +475,83 @@ _geoRadius(lat, lng, distance_in_meters)
 **Expected value**: An array of `attribute`s or `["*"]`
 **Default value**: `null`
 
-Returns the number of documents matching the current search query for each given facet. When `facets` is set, the search results object contains the `facetDistribution` field that shows the number of remaining candidates for each specified facet.
-
-This parameter can take two values:
+Returns the number of documents matching the current search query for each given facet. This parameter can take two values:
 
 - An array of attributes: `facets=["attributeA", "attributeB", …]`
 - An asterisk—this will return a count for all facets present in `filterableAttributes`
 
+When `facets` is set, the search results object contains the [`facetDistribution`](#facetdistribution) and [`facetStats`](#facetstats) fields.
+
 ::: note
 If an attribute used on `facets` has not been added to the `filterableAttributes` list, it will be ignored.
 :::
 
-[Learn more about facet distribution in the filtering and faceted search guide.](/learn/advanced/filtering_and_faceted_search.md#configuring-and-using-facets)
+#### `facetDistribution`
+
+`facetDistribution` contains the number of matching documents distributed among the values of a given facet. Each facet is represented as an object.
+
+#### `facetStats`
+
+When using the `facets` parameter, any matching documents with facets containing numeric values are displayed in a `facetStats` object. `facetStats` contains the numeric minimum (`min`) and maximum (`max`) values per facet for all documents matching the search query.
+
+If none of the matching documents have a numeric value for a facet, that facet is not  included in the `facetStats` object.
 
 #### Example
 
-Given a movie database, suppose that you want to know the number of `Batman` movies per genre:
+Given a movie ratings database, the following code sample returns the number of `Batman` movies per genre along with the minimum and maximum ratings:
 
-<CodeSamples id="faceted_search_facets_1" />
+<CodeSamples id="search_parameter_guide_facet_stats_1" />
 
-You would get the following response:
+The response shows the facet distribution for `genres` and `rating`. Since `rating` is a numeric field, you get its minimum and maximum values in `facetStats`.
```

Comment:
Shouldn't we add why it is useful to have this information for the user's standpoint?

e.g., Include a range slider component for the `rating` facet when creating a faceted search experience

Reply:
I added this to the guide as it seemed more appropriate